x/bank/types/codec.go

```diff
@@ -13,7 +13,6 @@ import (
 // on the provided LegacyAmino codec. These types are used for Amino JSON serialization.
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	legacy.RegisterAminoMsg(cdc, &MsgSend{}, "cosmos-sdk/MsgSend")
-	legacy.RegisterAminoMsg(cdc, &MsgMultiSend{}, "cosmos-sdk/MsgMultiSend")
 	legacy.RegisterAminoMsg(cdc, &MsgUpdateParams{}, "cosmos-sdk/x/bank/MsgUpdateParams")
 	legacy.RegisterAminoMsg(cdc, &MsgSetSendEnabled{}, "cosmos-sdk/MsgSetSendEnabled")
```

Comment:
You might also want to consider removing `MsgSetSendEnabled` as well,  I think the v50 version is `DefaultSetSendEnabled` or something

Reply:
Got it. And I think we can keep `MsgMultiSend`, thanks for pointing out!